JavaScript/inscript.js

```diff
@@ -0,0 +1,59 @@
+document.addEventListener('DOMContentLoaded', function() {
+    const emailInput = document.getElementById('email');
+    const emailError = emailInput.nextElementSibling;
+
+    const passwordInput = document.getElementById('password');
+    const passwordError = passwordInput.nextElementSibling;
+
+    const loginButton = document.getElementById('loginclick');
+
+    loginButton.disabled = true;
+
+    emailInput.addEventListener('focusout', () => {
+        const emailValue = emailInput.value.trim();
+        if (!emailValue) {
+            emailError.textContent = '이메일을 입력해 주세요';
+            emailInput.classList.add('error');
+
+        } else if (!/\S+@\S+\.\S+/.test(emailValue)) {
+            emailError.textContent = '잘못된 이메일입니다';
+            emailInput.classList.add('error');
+        } else {
+            emailError.textContent = '';
+            emailInput.classList.remove('error');
+        }
+    });
```

Comment:
P3:
아래 코드도 동일동작합니다.

```suggestion
    emailInput.addEventListener('focusout', () => {
        const emailValue = emailInput.value.trim();
        
        if (!emailValue) {
            emailError.textContent = '이메일을 입력해 주세요';
            emailInput.classList.add('error');
            return;
        }
        
        if (!/\S+@\S+\.\S+/.test(emailValue)) {
            emailError.textContent = '잘못된 이메일입니다';
            emailInput.classList.add('error');
            return;
        }

        emailError.textContent = '';
        emailInput.classList.remove('error');
    });
```